website/next.config.js

```diff
@@ -23,56 +21,85 @@ const env = {
 }
 
 const withNextra = nextra({
-  theme: './src/nextra-theme/index.tsx',
+  theme: './src/NextraLayout.tsx',
   search: false,
   codeHighlight: false,
   defaultShowCopyCode: false,
   readingTime: true,
   transformPageMap(pageMap) {
-    const locale = pageMap[0].data.slice(0, 2)
+    const locale = pageMap[0].route.slice(1, 3)
+    // TODO: remove this when crowdin will translate global.json
+    const t = (key) => {
+      try {
+        return translate(translations, locale, `global.sidebar.${key}`)
+      } catch {
+        return translate(translations, 'en', `global.sidebar.${key}`)
+      }
+    }
+
+    const metaFile = {
+      index: t('index'),
+      '---1': {
+        type: 'separator',
+      },
+      about: '',
+      network: t('network'),
+      sunrise: '',
+      billing: '',
+      glossary: '',
+      tokenomics: '',
+      arbitrum: t('arbitrum'),
+      '---2': {
+        type: 'separator',
+      },
+      '###1': {
+        type: 'heading',
+        title: t('subgraphs'),
+      },
+      'quick-start': '',
+      developing: t('developing'),
+      deploying: t('deploying'),
+      publishing: t('publishing'),
+      managing: t('managing'),
+      querying: t('querying'),
+      cookbook: t('cookbook'),
+      'release-notes': t('releaseNotes'),
+      '---3': {
+        type: 'separator',
+      },
```

Comment:
@benface now root _meta file will be translated by Crowdin, I remember you asked it in the past

Reply:
This is awesome @dimaMachina, thank you!